docs/content/direct/release-testing.md

```diff
@@ -0,0 +1,29 @@
+# Testing a new KubeStellar Release
+
+The following testing process should be applied to every KubeStellar new release in order to validate the release. All the tests should be done while the KubeStellar code is still under code-freeze and new code shouldn't be merged into the main branch until all tests are passed and the release is declared as stable.  
+In case the release tests fail (even one of them), the release should be marked as instable and a fix should be worked on ASAP. The KubeStellar code-freeze should be lifted only after all tests are passed and a stable releases was released.  
+To reduce the exposure of instable releases the update of the KubeStellar site [kubestellar.io](https://docs.kubestellar.io) should be done only once all release tests passed sussceffuly. 
```

Comment:
This language makes me wonder what you think about release candidates. Do you consider them to be releases? Note: that is the position take in release.md.

The word in English is "unstable", not "instable".

Every release is considered to be a second-class citizen when it is first created, in these ways: (a) the website does not consider it to be "latest" or "stable", and (b) GitHub calls it a "pre-release".

We have been using the concept of release candidate. I think that this document should not abandon it.

I think that we want to admit code changes that fix bugs while progressing from one release candidate to the next.

Reply:
I removed the references to "stable"  and just used release and regular releases